ihmc-perception/src/main/java/us/ihmc/perception/headless/HumanoidPerceptionModule.java

```diff
@@ -280,7 +280,7 @@ public void initializePerspectiveRapidRegionsExtractor(CameraIntrinsics cameraIn
    public void initializeHeightMapExtractor(HumanoidReferenceFrames referenceFrames, CameraIntrinsics cameraIntrinsics)
    {
       LogTools.info("Rapid Height Map: {}", cameraIntrinsics);
-      rapidHeightMapExtractor = new RapidHeightMapExtractor(openCLManager, referenceFrames);
+      rapidHeightMapExtractor = new RapidHeightMapExtractor(openCLManager, referenceFrames.getMidFeetUnderPelvisFrame());
```

Comment:
It seems like this doesn't need to be changed for this PR, can we leave this out?

Reply:
It was needed because we don't have an instance of HumanoidReferenceFrames to pass in, only the single frame. Since you only use that frame I didn't think it would be an issue.

Reply:
The reason you don't always have a HumanoidReferenceFrames is that PerceptionAndAutonomyProcess supports being run standalone in order to test sensor drivers and algorithms for development and debugging. In this case all the frames are just world frame and there is no robot model with which to create a HumanoidReferenceFrames.